lld/test/COFF/wrap-inherit-weakness.ll

```diff
@@ -0,0 +1,16 @@
+; REQUIRES: x86
+
+; Check that 'weak' attribute will be inherited to the wrapped symbols.
+
+; RUN: llc %s -mtriple x86_64-mingw -o %t.o --filetype=obj
+; RUN: ld.lld -m i386pep -shared -o %t.dll %t.o --entry= --wrap fn
+
+declare extern_weak dso_local void @__real_fn() nounwind
+declare dso_local void @fn() nounwind
+declare dso_local void @__wrap_fn() nounwind
```

Comment:
Is `__wrap_fn` also treated as weak, as `__real_fn` is weak, or what's the relation here? Wouldn't we otherwise have undefined references as the reference to `fn` gets redirected to `__wrap_fn`?